packages/angular/cli/models/architect-command.ts

```diff
@@ -267,10 +267,10 @@ export abstract class ArchitectCommand extends Command<ArchitectCommandOptions>
     }
 
     if (!project) {
-      throw new Error('No project specified');
+      project = '';
```

Comment:
Is this change meant so that Architect itself throws the error instead of throwing it here?

Reply:
The original CLI code didn't have either of the undefined checks (project/target).  They were needed by the stricter TS settings on devkit but apparently the absence of a project or target is actually acceptable at this point in the code.